src/wiktextract/extractor/en/example.py

```diff
@@ -0,0 +1,237 @@
+                if len(example_text) > 0:
+                    example = deepcopy(parent_example)
+                    # dialect and character variant tag
+                    for link_node in span_tag.find_child_recursively(
+                        NodeKind.LINK
+                    ):
+                        example["raw_tags"].append(
+                            clean_node(wxr, None, link_node)
+                        )
+                    example["text"] = example_text
+                    example["roman"] = roman
+                    example["english"] = translation
+                    example["raw_tags"].extend(roman_raw_tags)
+                    if len(ref) > 0:  # don't override parent quote-* template
+                        example["ref"] = ref
+                    clean_example_empty_data(example)
+                    results.append(example)
+
+    # no source, single line example
+    if not has_dl_tag:
+        roman = ""
+        raw_tags = []
+        for span_tag in expanded_node.find_html(
+            "span", attr_name="lang", attr_value="Latn"
+        ):
+            roman = clean_node(wxr, None, span_tag)
+        for span_tag in expanded_node.find_html("span"):
+            span_text = clean_node(wxr, None, span_tag)
+            if span_text.startswith("[") and span_text.endswith("]"):
+                raw_tags.append(span_text.strip("[]"))
+        translation = clean_node(
+            wxr, None, template_node.template_parameters.get(2, "")
+        )
+        literal_meaning = clean_node(
+            wxr, None, template_node.template_parameters.get("lit", "")
+        )
+        for span_tag in expanded_node.find_html("span"):
+            span_lang = span_tag.attrs.get("lang", "")
+            if span_lang in ["zh-Hant", "zh-Hans"]:
+                example_text = clean_node(wxr, None, span_tag)
+                if len(example_text) > 0:
+                    example_data = deepcopy(parent_example)
+                    example_data["text"] = example_text
+                    example_data["roman"] = roman
+                    example_data["tags"].append(
+                        "Traditional Chinese"
+                        if span_lang == "zh-Hant"
+                        else "Simplified Chinese"
+                    )
+                    example_data["english"] = translation
+                    example_data["literal_meaning"] = literal_meaning
+                    example_data["raw_tags"].extend(raw_tags)
+                    clean_example_empty_data(example_data)
+                    results.append(example_data)
+    return results
+
+
+ZH_X_TAGS = {
+    "trad.": "Traditional Chinese",
+    "simp.": "Simplified Chinese",
+}
+
+
+def clean_example_empty_data(data: ExampleData) -> ExampleData:
+    raw_tags = data.get("raw_tags", [])
+    new_raw_tags = []
+    for raw_tag in raw_tags:
+        if raw_tag in valid_tags:
+            data["tags"].append(raw_tag)
+        elif raw_tag in ZH_X_TAGS:
```

Comment:
This might be better first; if someone adds "trad." or "simp." to tags.py, this branch would break, and no one would know.

Reply:
I mean, `elif raw_tag in ZH_X_TAGS` should be first and `raw_tag in valid_tags` should be in `elif`.